packages/quantic/force-app/main/default/lwc/quanticUserAction/quanticUserAction.js

```diff
@@ -0,0 +1,103 @@
+import {LightningElement, api} from 'lwc';
+import {DateUtils} from 'c/quanticUtils';
+// @ts-ignore
+import actionTemplate from './templates/action.html';
+// @ts-ignore
+import viewActionTemplate from './templates/viewAction.html';
+import ticketCreated from '@salesforce/label/c.quantic_TicketCreated';
+import emptySearch from '@salesforce/label/c.quantic_EmptySearch';
+
+/** @typedef {import("coveo").UserAction} UserAction */
+
+const icons = {
+  SEARCH: 'utility:search',
+  CLICK: 'utility:file',
```

Comment:
wouldn't `pop_in` or `target_mode` makes more sense than file for CLICK events? Maybe we should check with UX what they think. Personally, I think file looks more like the generally accepted icon for `copy-to-clipboard` rather than a click

Reply:
its unfortunate there are no better icons for this though

Reply:
Indeed the icon we currently using can be seen as the copy to clipboard icon but like you also mentioned there no really endless choices to pick from.

`pop_in` or `target_mode` seems like icons that can be used to say that something can be clicked to be opened in a new window, but currently the feature to open the clicked document is not available yet.

so I think for now without being able to open the document, the current icon is our best choice